samples/gazell/gzll_ack_payload/device/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
+  name: Gazell ACK payload device sample
+tests:
+  samples.gazell.gzll_ack_payload.device.build:
+    build_only: true
+    build_on_all: true
```

Comment:
Please remove this line. It tells twister to try to build on every existing platform in zephyr (~400) which is pointless.

Reply:
Will do.